src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,226 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.exception.emptyException;
+import duke.exception.nonMatchException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+//import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+
+    public static final int MAX_TASK_NUM = 100;
+    public static Task[] taskArray = new Task[MAX_TASK_NUM];
+    //ArrayList<Task> tasks = new ArrayList<>();

```

Comment:
This seems like past code that is commented out as it is no longer needed. You may want to delete this - do make full use of version control to keep track of dead code.